setup.py

```diff
@@ -32,9 +32,13 @@
     packages=['edx_rest_framework_extensions'],
     install_requires=[
         'django>=1.8.9,<2.0',
-        'djangorestframework>=3.2.3,<4.0.0',
+        'djangorestframework==3.6.3',
```

Comment:
It was necessary to pull in the edx fork of DRF as the latest version of DRF is not compatible with 1.9  This keeps this library in line with edx-platform